example/annotation_surveyor.dart

```diff
@@ -80,17 +80,17 @@ void main(List<String> args) async {
   print('  aliases (when appropriate).');
 }
 
-int dirCount;
+int dirCount = 0;
 
 /// If non-zero, stops once limit is reached (for debugging).
-int _debugLimit; //500;
+int? _debugLimit; //500;
```

Comment:
The comment says "if non-zero", so consider initializing it to zero (or fixing the comment if it's wrong). If `null` is being used as a sentinel value, consider adding a `useDebugLimit` boolean instead.